workarea-one_theme.gemspec

```diff
@@ -7,26 +7,27 @@ require "workarea/one_theme/version"
 Gem::Specification.new do |s|
   s.name        = "workarea-one_theme"
   s.version     = Workarea::OneTheme::VERSION
-  s.authors     = ["Jake Beresford"]
-  s.email       = ["jberesford@weblinc.com"]
+  s.authors     = ["Curt Howard"]
+  s.email       = ["choward@workarea.com"]
   s.homepage    = "https://github.com/workarea-commerce/workarea-one-theme"
-  s.summary     = "Minimal, versatile theme for Workarea"
+  s.summary     = "Minimal, versatile theme for workarea ecommerce"
   s.description = "A slick, polished, theme for smaller catalogs."
 
   s.files = `git ls-files`.split("\n")
 
-  s.add_dependency "workarea", "~> 3.x", ">= 3.4.x"
-  s.add_dependency "workarea-theme", "~> 1.x"
-  s.add_dependency "workarea-product_quickview", ">= 1.2.x"
-  s.add_dependency "workarea-slick_slider", "~> 1.x"
-  s.add_dependency "workarea-blog", '~> 3.x', ">= 3.2.x"
-  s.add_dependency "workarea-content_search", "~> 1.x"
-  s.add_dependency "workarea-filter_dropdowns", "~> 3.x"
-  s.add_dependency "workarea-gift_cards", "~> 4.x"
-  s.add_dependency "workarea-share", "~> 1.x", ">= 1.2.x"
-  s.add_dependency "workarea-reviews", "~> 3.x"
-  s.add_dependency "workarea-super_hero", "~> 2.x", ">= 2.1.x"
-  s.add_dependency "workarea-wish_lists", "~> 3.x"
-  s.add_dependency "workarea-swatches", "~> 1.x"
-  s.add_dependency 'workarea-search_autocomplete', '~> 1.x'
+  s.add_dependency 'workarea', '>= 3.5.0'
```

Comment:
Any reason to not allow prereleases here anymore?

Reply:
Like I said in the NVY PR, I want to see if I can solve the prerelease problem with the Upgrade plugin using `bundle outdated --pre`. When we push up a beta for a new Workarea version it won't be compatible with the theme OOTB, so locking it down to stable versions feels safer to me. If it turns out that this problem can't be solved with the Upgrade plugin then I'll go back in these themes and enable pre-releases in each.